scripts/install/config.default.sh

```diff
@@ -41,7 +41,7 @@ SPARTACUS_PROJECTS=(
         )
 
 SPARTACUS_REPO_URL="https://github.com/SAP/spartacus.git"
-BRANCH='develop'
+BRANCH='release/5.0.x'
```

Comment:
That's ok, just remember not to port this change to develop when ng 14 is added there.